scripts/publish

```diff
@@ -49,6 +49,7 @@ fi
 
 if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
     rm -rf dist
-    python setup.py sdist bdist_wheel
+    pip install build twine
+    python -m build --sdist --wheel
```

Comment:
Nice, I solved this another way for sentinel/landsat, but this is better - should update the template script

Reply:
done did it! https://github.com/stactools-packages/template/commit/939a7766acbc28925c326bf108d2f64d21fb3916